samples/bookinfo/istio.io_tutorial/route-rule-reviews-80-20.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017,2018 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
no need to add the copyright for yaml template

Reply:
What is the rationale though? The deployment YAML files have copyright, e.g. https://github.com/istio/istio/blob/master/samples/bookinfo/kube/bookinfo.yaml